CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Print absolute filepaths as relative again (as with 2.1 and before) if they
   are below the current directory (The-Compiler, #246).
 * Run tests and add PyPI trove for Python 3.10 (chayim, #266).
+* Add support for `del` keyword to mark variable as used for function/methods overrides
```

Comment:
Do you want to add your name and the PR number here?

Reply:
Added